### ISSUE TITLE
Apptimize integration requires older version of segment analytics

### ISSUE THREAD

**Issue by msalmanST** [segment-integrations/analytics-ios-integration-apptimize#1]
Hi, 

When I try to add Apptimize integration in my code I face an issue of segment version being used in my code, please see the error below.

```
[!] Unable to satisfy the following requirements:

- `Analytics (= 3.3.0)` required by `Podfile`
- `Analytics (~> 3.0.7)` required by `Segment-Apptimize (0.1.0)`
```

So should I wait for the new release of segment-apptimize integration or should I downgrade my segment version from 3.3.0 to 3.0.7 ?

Thanks.

**Comment by f2prateek** [comment]
Thanks for reporting, fixed in 0.2.0!

**Comment by f2prateek** [comment]
Sorry, actually I realized we gave the ownership of the pod to Apptimize (https://github.com/Apptimize/analytics-ios-integration-apptimize).

You'll have to contact their team to make this fix https://github.com/segment-integrations/analytics-ios-integration-apptimize/commit/3ead6398461ed0fe1efc51f8fd380361c814d8f6 in their github repo.
